recipes/cmaps/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "cmaps" %}
+{% set version = "1.0.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "644ad7b5d45923effe275be02622ba0290d8a2f06de7fc1e2e0c90df66f20c13"
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - matplotlib
+    - numpy
+    - pip
+    - python
+  run:
+    - matplotlib
+    - numpy
+    - python
+
+test:
+  imports:
+    - cmaps
+
+about:
+  home: "https://github.com/hhuangwx/cmaps"
+  license: "LICENSE"
+  license_family: "GPL v3.0"
+  license_file: ""
```

Comment:
The license file is not in the pypi tarball, so you can add the file from the repo here, https://github.com/hhuangwx/cmaps/blob/master/LICENSE, and use `license_file: LICENSE`.

Reply:
Done